pygotham/frontend/profile.py

```diff
@@ -20,7 +20,19 @@
 def dashboard():
     """Return the user's dashboard."""
     # TODO: Optionally, old proposals should be shown in a read-only mode.
-    talks = Talk.query.current.filter(Talk.user == current_user)
+    talks = db.session.query(
+        Talk,
+        Speaker,
+    ).join(
+        'speakers'
+    ).filter(
+        Speaker.user == current_user
+    ).filter(
+        Speaker.declined_at is None
```

Comment:
I think this wants to be `.is_(None)` or `== None`, `is` cannot be overloaded so sqla uses a method or `==` to check for `NULL`